src/commands/scanner/run/dfa.ts

```diff
@@ -68,6 +68,11 @@ export default class Dfa extends ScannerRunCommand {
 			description: messages.getMessage('flags.sfgejvmargsDescription'),
 			longDescription: messages.getMessage('flags.sfgejvmargsDescriptionLong'),
 			env: 'SFGE_JVM_ARGS'
+		}),
+		'pathexplimit': flags.integer({
```

Comment:
New parameter

Reply:
The convention is to use kebab-casing. Could we call this `path-exp-limit`?